src/types/ui.types.ts

```diff
@@ -1,12 +1,25 @@
 export type UiDef = UiDefinition | LegacyUiDefinition;
 
+export interface SfUIDefinition {
+  Id?: string;
+  VELOCPQ__ModelId__c: string;
+  VELOCPQ__Default__c: boolean;
+  VELOCPQ__SourceBlob__c?: string;
+  VELOCPQ__SourceDocumentId__c: string;
+  VELOCPQ__ModelVersion__c: string;
+  VELOCPQ__ReferenceId__c: string;
+}
+
 export type UiMetadata = Omit<UiDefinition, 'children'> & {
   children: string[];
 };
 
 export interface UiDefinition {
   name: string;
   version: number;
+  default?: boolean;
+  primary?: boolean;
+  referenceId: string;
```

Comment:
Could you please add another type just to not mix UIDefinition with SF entity:

```
export interface UIDefinitionContainerDTO {
  id: string;
  modelId: string;
  sourceBlob: string;
  version?: number;
  referenceId: string;
}
```

UiDefinition itself shouldn't be changed.

Reply:
Done